_entries/02-03 challenge2-2.md

```diff
@@ -75,10 +84,7 @@ spec:
               cpu: "500m"
           env:
           - name: MONGOHOST
-            valueFrom:
-              secretKeyRef:
-                name: mongodb
-                key: mongoHost
```

Comment:
This won't work with the current DevOps pipeline (see: https://github.com/Azure/azch-captureorder/blob/master/azure-pipelines.yml#L69)